translations/resources.tex/pl.po

```diff
@@ -68,6 +68,23 @@ msgid ""
 "  \\includegraphics[width=0.8\\linewidth]{\\images/valuables.png}\n"
 "  \\footnotesize\\textcolor{darkcandyapplered}{\\textit{\\textbf{Valuables \\phantom{Materials}}}}"
 msgstr ""
+"\\hypertarget{Resources}{W grze} istnieją trzy rodzaje \\index{Zasoby} zasobów: Złoto \\includesvg[height=10px]{\\svgs/gold.svg}, Budulce \\includesvg[height=10px]{\\svgs/building_materials.svg} oraz Kosztowności \\includesvg[height=10px]{\\svgs/valuablegreater.svg}.\n"
+"Zasoby będziesz wydawał w trakcie gry na rozbudowę \\hyperlink{Town}{Miasta}, Rekrutację \\hyperlink{Units}{Jednostek} i zakup \\hyperlink{spells}{Zaklęć}.\n"
+"Zazwyczaj będziesz je pozyskiwał z \\hyperlink{Mines}{Kopalni oraz Osad}, które \\hyperlink{Categories}{Oflagowałeś}, a także zagrywając Karty i rzucając Kośćmi Zasobów \\includesvg[height=10px]{\\svgs/resource_die.svg}.\n"
+"Za każdym razem, gdy produkcja Zasobów gracza zostanie zwiększona lub zmniejszona, należy przesunąć Kostkę tego Zasobu na jego Torze Produkcji o odpowiednią liczbę pól.\\par\n"
```

Comment:
"Za każdym razem, gdy produkcja Zasobów Gracza zostanie zwiększona lub zmniejszona, należy przesunąć Kostkę tego Zasobu na jego Torze Produkcji o odpowiednią liczbę pól.\\par\n"

Reply:
Gracz to nie jest komponent.